autoload/spaceduck.vim

```diff
@@ -0,0 +1,10 @@
+" Helper function that takes a variadic list of filetypes as args and returns
+" whether or not the execution of the ftplugin should be aborted.
+func! spaceduck#should_abort(...)
```

Comment:
I'm not sure I follow the statements here; the should_abort function (to my knowledge) is basically just a check that looks if the current file is a known syntax, and if coloring is enabled, and either highlights that or not.

For example, my markdown file looks like the one in the screenshot above, with no plugins at all installed, just using the standard neovim/vim markdown highlighting.

If I install `vim-polyglot` and then also do a `let g:polyglot_disabled = ['markdown']` before its loaded, the file just looks identical to the above, which is the default syntax highlighting enabled by (neo)vim.

If I remove that polyglot_disabled line, it then changes the highlighting of the markdown file (since I didn't add any of the groups from the one that vim-polyglot uses, which looks like plasticboy

Reply:
Oh my apologies, I mistook what the function would actually do. You're right.